README.md

```diff
@@ -0,0 +1,117 @@
+# Project Marley
+
+Marley is a person recognition service that is run locally on a mini-PC at the entrance to the Dynabyte office. Users walk into the office and are greeted by Marley, which may become like a simplified automated receptionist in a future release.
+
+## Table of Contents
+- [Use Cases](#use-cases)
+- [Structure](#structure)
+
+## Use Cases
+PREDICTION <br>
+The client application takes images and tries to check if a face can be found in each image, and if so predict which person is in the image. If Marley recognizes a person, it will greet them by name. If not, that person will still be greeted without a name and asked to register.
+
+REGISTRATION <br>
+If a person chooses to register, face encodings are saved along with their name which Marley can then use to recognize them. If no face is found, then no greeting is displayed.
+
+DELETION <br>
+A person can choose to no longer be recognized by Marley, in which case their personal data and face encodings will be removed.
+
+## Structure
+<pre>
+Client application (React, typescript)
+🠗			 🠕
+Person Recognition (Java rest API, Spring Boot)		⮌	PostgreSql - person info
+	🠗	🠕
+Face Recognition (Python Flask rest API)	        ⮌	MongoDB - face encodings
```

Comment:
It would be good if you could add what (JSON schema explanation) are stored in MangoDB.